corc-cascading/src/main/java/com/hotels/corc/cascading/OrcFile.java

```diff
@@ -408,6 +410,13 @@ public SourceBuilder schemaFromFile() {
     public SourceBuilder searchArgument(SearchArgument searchArgument) {
       checkExisting(this.searchArgument, "a search argument");
       checkNotNull(searchArgument, "searchArgument");
+      List<PredicateLeaf> leaves = searchArgument.getLeaves();
+      for (PredicateLeaf leaf : leaves) {
+        if (Type.TIMESTAMP.equals(leaf.getType())) {
```

Comment:
I'd like to have figured out what the underlying issue is but lack of time and low usage of Corc means this is as good as it's going to get for now :(